system_modules/napmath/src/mathutils.cpp

```diff
@@ -125,52 +125,6 @@ namespace nap
 		}
 
 
-		float smoothDamp(float currentValue, float targetValue, float& currentVelocity, float deltaTime, float smoothTime, float maxSpeed)
```

Comment:
By moving the definition into the header you define it in every compilation unit. Because the implementation isn't 'small'  this could add a considerable size to the overall compiled libraries and executables. That is the reason we explicitly define one for float and double - to minimize overhead. I would recommend moving the the template definition into the cpp again and define it there.